src/resource/resource_manager.hpp

```diff
@@ -8,7 +8,7 @@
 
 class ResourceManager {
    public:
-    static std::unordered_map<std::string, std::shared_ptr<ResourceBase>> resources;
```

Comment:
driveby: this should never have been static, as it made replacing the `resource_manager` during a `refresh` call not actually do anything.